src/lib/components/Event/Event.svelte

```diff
@@ -37,7 +38,7 @@
       <Link
         class="display-block"
         href={date && event?.slug
-          ? `/about/events/event/${date.toISOString().split("T")[0]}-${event.slug}`
+          ? `/about/events/event/${constructEventSlug(date, event.slug)}`
           : undefined}
```

Comment:
This wasn't created by you, but just something that may be a matter of opinion for setting things as `undefined`. From what I've heard, setting `undefined` vs using `null` is not recommended because its supposed to represent a value that is unassigned. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined